client/galaxy/scripts/ui/scroll-panel.js

```diff
@@ -12,7 +12,7 @@ $.ui.plugin.add("draggable", "scrollPanel", {
             panel_pos = panel.position(),
             panel_w = panel.width(),
             panel_h = panel.height()
-            viewport = panel.parent();
+            viewport = panel.parent(),
```

Comment:
Looks like this file needs a little more love; @xiemaisi should LGTM have added a comma after the line above?